remote/advancedcontainers/sharing-git-credentials.md

```diff
@@ -85,6 +85,10 @@ If you do not have GPG set up, you can configure it for your platform:
   * Register a `pinentry` GUI in your WSL distro. `echo pinentry-program /mnt/c/Program\ Files\ \(x86\)/Gpg4win/bin/pinentry.exe > ~/.gnupg/gpg-agent.conf`
   * Reload the `gpg` agent in WSL. `gpg-connect-agent reloadagent /bye`
 
+> **Note**
+> For Windows user, the gpg signing key must be setup using the Windows GUI or cli (powershell/cmd) and not in the Git Bash.
+> For some unknown reason, the DevContainer can't access the gpg keys configured in Git Bash even though it is in your `~/.gnupg/` folder, accessible in the Windows explorer.
```

Comment:
Actually the reason is that the extension uses `gpg` on the Windows `PATH` to discover the running `gpg-agent` and Git Bash's `gpg` command is not on the `PATH`. Not sure if adding Git Bash's `gpg` to the `PATH` would work. I suggest to omit the "for some unknown reason" part.

Reply:
Corrected the sentence.
Adding the command to the `PATH` might be a solution.